portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,11 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+console.log("hello world");
+
+async function getRandomQuoteUsingAsyncAwait() {
```

Comment:
Uber Nit:
I suggest changing to just getRandomQuote(). The caller of the function should not care about the implementation.

Reply:
makes sense! done!